app/controllers/integrations/feature_flags_controller.rb

```diff
@@ -5,6 +5,7 @@ def index
         {
           name: feature_name.humanize,
           active: FeatureFlag.active?(feature_name),
+          type: FeatureFlag::FEATURES[feature_name].type.humanize,
```

Comment:
Does this need to be humanized given it's only for reading by computers?

Reply:
I went with humanize to be consistent with the name which is also humanized

Reply:
actually it doesn't matter, will remove